flow/size.yaml

```diff
@@ -16,6 +16,10 @@ components:
               x-field-uid: 2
             random:
               x-field-uid: 3
+            weight_pairs:
+              x-field-uid: 4
+            quad_gaussian:
```

Comment:
Is this a placeholder or should it be removed as there is no property for this choice?

Reply:
Missed this one - Removed.